### PR TITLE
minor: Simplify flashing when core/worker on same machine

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Sending images is slow and pointless, if the core and worker components both live in the same machine. This newer flashing implementation is not final or tested but it's my shot at it. I still need to figure some things out but it's looking good. Stay tuned. 

﻿Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

### CODE REVIEW

File: worker/lib/index.ts [hunk 1]
```diff
@@ -286,7 +286,11 @@ async function setup(): Promise<express.Application> {
 
 			try {
 				worker.on('progress', onProgress);
-				await worker.flash(req);
+				console.log("Request is")
```
Comment (rcooke-warwick) [inline worker/lib/index.ts:289]: These console log debug statements can probably be removed now right?

File: worker/lib/index.ts [hunk 2]
```diff
@@ -286,7 +286,11 @@ async function setup(): Promise<express.Application> {
...
+				console.log(req)
+				console.log(req.body.imagePath)
+				console.log(JSON.stringify(req))
+				// await worker.flash(req.);
```
Comment (rcooke-warwick) [inline worker/lib/index.ts:293]: the actual flashing is commented out

File: worker/lib/workers/qemu.ts [hunk 3]
```diff
@@ -20,14 +20,15 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
...
-	private internalState: Leviathan.WorkerState = { network: {wired: 'enp0s3'} };
+	private internalState: Leviathan.WorkerState = { network: { wired: 'enp0s3' } };
 	private screenCapturer: ScreenCapture;
 	private qemuOptions: Leviathan.QemuOptions;
 
 	constructor(options: Leviathan.Options) {
 		super();
 
 		if (options != null) {
+			console.log("Options ARE:")
```
Comment (rcooke-warwick) [inline worker/lib/workers/qemu.ts:31]: more debug comments to be removed

File: worker/lib/workers/qemu.ts [hunk 4]
```diff
@@ -78,33 +79,33 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
...
-				[destination],
-				(_destination, error) => {
-					reject(error);
-				},
-				(progress: sdk.multiWrite.MultiDestinationProgress) => {
-					this.emit('progress', progress);
-				},
-				true,
-			);
+			// const source = new sdk.sourceDestination.SingleUseStreamSource(stream);
```
Comment (rcooke-warwick) [inline worker/lib/workers/qemu.ts:86]: We can remove this commented out block right?

**Comment by klutchell** [comment]
We may want to put this on hold if we are considering moving core to the client.

**Pull request closed without merging**